docs/shared/tutorials/angular-standalone.md

```diff
@@ -517,7 +517,7 @@ More info can be found in [the integrate with editors article](/getting-started/
 
 Run the following command to generate a new "hello-world" component. Note how we append `--dry-run` to first check the output.
 
-```{% command="npx nx g @nx/angular:component hello-world --directory=src/app/hello-world --standalone --dry-run" path="myngapp" %}
+```{% command="npx nx g @nx/angular:component src/apps/hello-world/hello-world --standalone --dry-run" path="myngapp" %}
```

Comment:
The double `hello-world/hello-world` doesn't match the library and application generators.  The project generators make a folder at the location set by the path.  The component generator makes a set of files at the location set by the path.  It makes it hard for people to remember to double the generator name in the path, but only use it a single time in a project name.

Reply:
We're changing this to keep things consistent between all the artifact generators (like component).